src/collections/_documentation/enriching-error-data/context.md

```diff
@@ -8,17 +8,37 @@ example_extra_key: character_name
 example_extra_value: 'Mighty Fighter'
 ---
 
-{% include platforms/event-contexts.md %}
+Additional data attached to an event can be predefined or custom.
 
-## Capturing the User
+[Predefined data](#predefined-data) is data that Sentry recognizes and uses to enhance the user experience. For example, Sentry's concept of a user lets us show how many unique users are affected by an issue. Sentry automatically captures predefined data where possible, such as information about the browser, HTTP request, and OS in the browser Javascript SDK.
 
-Sending users to Sentry will unlock a number of features, primarily the ability to drill down into the number of users affecting an issue, as well to get a broader sense about the quality of the application.
+[Custom data](#custom-data) is arbitrary structured or unstructured extra data you can attach to your event.
 
-Capturing the user is fairly straight forward:
+## Tags & Context
+
+Whether predefined or custom, additional data can take two forms in Sentry: tags and context.
+
+Tags are key/value string pairs that are indexed and searchable. They power UI features like filters and tag-distribution heat maps.
```

Comment:
Can we not say 'heat map'. The visualization we use for tag distributions is not a heat map.

Reply:
👍

Reply:
@MimiDumpling could we change this to "tag-distribution maps"? (mark if you have a different suggestion let us know)

Reply:
Will do. Thanks!